pkgs/applications/editors/vis/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, unzip, pkgconfig, makeWrapper, ncurses, libtermkey, lpeg, lua }:
+
+stdenv.mkDerivation rec {
+  name = "vis-${version}";
+  version = "git-2016-04-15";
+
+  src = fetchurl {
```

Comment:
@joachifm updated with the corrected packages name using -nightly, and the fetchFromGithub.